### DISCUSSION TITLE
How to drag and drop objects

### DISCUSSION THREAD

**Discussion by Adaptoid17** [spatialsys/spatial-unity-sdk#13] [Help] [answered]
I am making a classroom-like environment and It will be about math where the students have 16 balls and drag and drop those balls to make a sum that leads to 16.
I am confused on how to implement the drag and drop so if anyone has any idea please help.

**Comment by AlanSpatial** [comment, marked as answer]
The only built-in drag and drop feature is for the objects you upload directly to spatial. So like how you can upload an image or model than drag it around. This is not available for gameObjects you define in a toolkit scene unfortunately.

In order to create a game around this you'll need to program the mechanic yourself. [This video](https://www.youtube.com/watch?v=yalbvB84kCg) has a pretty simple example of how you could get started.
